oracle/oracle_test.go

```diff
@@ -29,14 +30,15 @@ func TestAll(t *testing.T) {
 		t.Skipf("skipping Oracle tests as environment is not set...\n")
 	}
 
-
 	compartmentID, _ := cloudops.GetEnvValueStrict(fmt.Sprintf("%s", envCompartmentID))
 	availabilityDomain, _ := cloudops.GetEnvValueStrict(fmt.Sprintf("%s", envAvailabilityDomain))
+	kmsKeyID, _ := cloudops.GetEnvValueStrict(fmt.Sprintf("%s", envKmsKeyID))
```

Comment:
I am curious --
* why are they using `GetEnvValueStrict(fmt.Sprintf("%s", XXX))`
* isn't that the same as `GetEnvValueStrict(XXX)`

If the point was to "make a _copy_ of the string" --  this is not necessary, as strings are immutable  (once created, can't be changed)